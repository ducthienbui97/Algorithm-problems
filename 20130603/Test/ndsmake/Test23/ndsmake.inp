2350
-856075321 -386791847 213474084 829880663 775173049 -538748897 -493982729 -735974072 -982112458 -560148585 -682690401 -434475436 19263232 -365986146 -448136429 -942900816 -193782673 -632494071 -376356174 -202355141 896778207 -5946740 3937686 -390338062 -805313657 682639926 155341801 138635146 528074692 961601642 869533543 -458924997 141987365 -42781239 472471042 -925702087 966717506 970086683 -5101183 672988019 -742567273 -665439431 -146339911 -991653948 666813937 118436203 -444767208 304868990 693891130 128207894 124224002 -734519225 749322040 -668916316 -681402847 -273892117 -951657518 -315569712 -565976989 -380740592 -360185162 -976642756 679098464 -38116948 617495101 342293393 50555500 519680219 -823461500 -902897883 198127346 -415296360 -792601698 618797382 64071162 -698271208 -747928282 -479051574 139704004 134976888 226443106 -730569816 -111135791 945618870 -327054287 432611773 -644318851 12485125 394114597 571315476 -846482457 851249307 -68169823 854553794 710208395 748954735 -165093160 980003616 -906608561 820301364 823422578 -198060783 731436301 -387744722 -478244993 618688225 646171829 606125582 249828269 598177210 50770530 154504009 -782251280 -957277599 -62528979 -520691213 -617058839 -548335814 456770020 -473294718 129595884 322624263 181207186 -803896716 775941905 95386890 -271030501 416686352 492820568 -320671020 731982237 886986267 -743519604 826780914 -733296589 -479753718 19877855 767943250 -478481717 -191441586 739663236 564951830 -268928061 569791081 -928678550 -532654441 17828002 -766778047 359768449 -586306507 -854594189 -618875730 362582899 -389238981 -573942679 -926241728 -55878621 -266857380 42453329 505289445 -690676302 -775501000 926700743 699747430 971506520 303601210 355252719 -243955846 225875521 -166720347 -868387758 -913319514 5704832 612259870 979876318 -982940817 745610091 627109968 323053796 -927472217 720406283 360969394 -884309830 732889090 158091891 -271769483 -646973389 -143680181 903288691 737363495 -368337816 -726830459 890485176 399605712 -335515911 -35371564 -800791445 -495367953 -918662783 76044812 -633276426 -105400789 48089301 -302177818 220363770 693143390 -621603465 -581478165 527644843 -179398508 41280072 545443759 -279096657 -702360679 575446091 512322318 156517332 -257859543 247503666 893362108 -536467994 282880130 -242913821 -251364154 160686088 807729543 -963577442 128707240 992135385 -226684950 -566117372 -544845956 -699507298 -783658733 166367050 -895010901 31851023 413675545 518273049 -339175488 359314107 -1040147 -439109882 753342808 -774556648 -308896874 -846596183 -674691786 -303306709 979751318 -61794468 -521634701 24646864 747313004 -556608814 -476248921 771367987 219783121 -31851470 839912603 -75929240 873163597 429006620 798433296 475849929 961867299 10763688 -894898600 284015151 -375255955 -138502478 506242174 -498525302 773003309 491729713 456898637 103196976 234132612 -383157282 981266936 -985974448 542849207 564344025 -177969294 69443617 -631680923 916837651 745769307 820388195 -111519417 -248816814 -94326639 15613484 432426200 -430138474 -855177543 -102407411 701296287 209797974 688023567 -233438453 56256749 576389539 -616460467 -529413249 69303571 336613669 -531719077 -454324457 669340942 714289454 -65864169 505865655 64145910 -194863412 645824017 156297087 320400593 -837714292 500503590 -682122915 -455551812 -548200352 -203941443 284360892 466347444 -963059813 -75680098 189499482 759921154 -347963112 -9371286 910049335 -857114559 -509931141 625536443 553123853 -965335665 229097944 -596842355 -875104438 -832733489 810642965 -623544681 4252174 -335727104 803127237 199151051 586984955 286198237 606970117 -237719415 -985826009 713141518 -761841818 132221471 -910068319 -449672511 -433880216 -539146642 -738285580 350195232 -284605511 -195923968 -51759312 556574117 864530240 -647608380 95642228 665351902 348230175 -446292480 -675153977 713912661 -574485938 -751221366 -86044083 -950466942 -755963594 450588940 -437987208 411661771 36198761 979002428 66419002 -776618028 361042584 -763059079 265222613 577474543 906986630 -853712512 -324880829 -265377742 -282770535 -696246258 710641244 906579314 276089746 -130616000 48554370 -791523733 -580263863 103400100 438604669 261075456 -745844185 -503912698 -777785018 806151416 -770852034 95312202 -315733288 986136607 -14700371 45371462 -3514980 -164735770 697606387 -599747006 211890370 -233860559 -837930158 -715306758 -119225756 -431270808 52185982 176891143 215362614 -823548757 804347362 544990557 315038188 386300194 -61112695 -532948755 513149737 -898078607 51328175 173273195 -161318114 336716363 -287851291 176137770 -978475065 -351527138 949027803 -794410461 -529640770 917248236 160056269 -671278060 124351931 -962265286 817155725 45129157 403145897 -968837088 661728809 -112466075 -813861019 -150069119 629499448 -982312940 488273173 115203413 -177619002 495082618 897489699 793882658 -549049895 369949141 5202058 139516064 -997062257 213024907 -110662403 -966824649 -589598516 498938772 -237972338 -955871207 999990700 859342883 715069573 325537607 43120705 386403227 409150426 487361975 -266865142 951856764 -149685784 -50253130 31713089 404931158 -43603047 148523878 -801179010 288385336 541635491 -341077347 542875511 2934656 -426561226 826882472 927439792 677756370 -766409546 -721747604 898886259 -403554551 373219105 487988159 718244959 85066747 -176705077 814989906 103761273 -340189741 -473825885 -374673741 654020061 614390855 229510486 519883346 484713517 -325642277 -764568286 1880696 -306214689 38141548 -401351092 -613188583 286752936 565927098 225144277 -941315890 734381004 143130411 -285647822 -792536709 752778937 -420908051 -161360500 238672922 -162985288 -163590622 -167564800 -805589180 -244762545 -186176967 -385036145 795016144 -563305161 -203692712 89353897 -994556270 -934463499 -187852299 183078481 429472837 -112100584 236276973 802527619 -43699698 733645643 -46797652 347087266 554213457 -97463660 251900279 -285825514 -256533387 212605735 -101880148 -684764967 338278554 -933916751 804415461 -998624768 995583537 -827369996 -795098953 923280046 -645343775 584290229 499454188 -940101877 -813749660 378627366 -37327243 160447117 772054924 -171215541 -993240713 930444680 356839364 502379200 393103809 743454248 -606855392 260625146 -601466389 -42586502 926067968 667561894 725240327 -922046044 171636315 331160621 590328839 -512107979 -57265491 -67579559 -110564437 402031511 831788415 818819345 10855460 -872339295 -618217840 -334838720 -512092854 520935753 -940227939 -318702985 305699191 470922962 686007779 -877107970 870750964 -544978739 856779751 -281084853 954197121 -617902773 -709245142 -182000231 -137750524 202035057 -991089334 522070494 -509490503 -83855710 -822596634 -123077800 703977847 -924935799 871170227 118745659 -869580281 -705080649 -654932098 -645010282 356155267 -33262942 -82065638 257125257 -3824645 483366444 -616424710 768925837 228420643 -68122292 466291470 149995357 832589024 -304481153 789993064 323119076 122601026 -203228675 702491912 -326340494 -685178564 -711692493 -438839363 842037762 273829803 75753820 617043455 672689072 -668211025 -566948113 729128425 -33620292 98204044 -720166794 993796509 -745682771 864221728 32011389 -94830251 804685202 337549057 -278626139 12522503 -955808668 484762418 -931651112 -397045580 -761841785 801450882 -774310505 -4406706 832637807 484140418 838317678 -631648188 -559037089 -388312031 105033097 -566082133 -12334003 -307225626 -674469023 -650075894 708828642 448667848 897587599 875943618 154980765 -961239123 168810433 -373740521 21991649 -812424095 787170787 242154707 496258505 -941498999 -796716128 -76113489 -609264321 -549865924 733269131 -926508467 -571544791 -829618625 324047796 801705957 -52573545 -740455826 -69039527 467063788 256014440 -216042836 232201840 308195239 62908170 823393227 721325782 -375214848 343410941 373264249 235359302 488580890 713943119 544337711 350827851 699349648 42074493 -765123540 148431335 -54545832 -554759209 -274072297 543110895 -379174137 327096576 858295381 511850633 -509414521 -261269366 977884423 -737556720 450736328 890772707 -59591011 -354173598 -205473291 -545872098 631656590 940509103 290857152 -269451236 159508540 -240709438 837151085 -248184589 -175202346 -345254856 749541123 720405434 440977215 -666898048 500441328 651672939 308233612 67846751 102290383 -562263347 94424632 396630382 930385824 726084682 874116148 859163837 666032080 112022293 992058475 610452914 -404068918 -785845877 570409843 845401402 -966949310 -20418635 106472686 536520986 76965354 -329466237 -49808285 681039361 -57204846 470534261 226998133 -998488059 -44016163 637841529 321703603 -107789423 -895766904 303268258 101413394 -520563045 -440126353 -246734418 877991947 98884829 -228587825 922404406 877883621 51049165 237176037 705099366 -340192482 72528456 761215318 363132356 598428255 -73056542 -812181641 -876130532 -241450302 -620833652 -651132632 -499935311 -494368264 642245701 -625685400 -162011734 -694381769 -347550449 991599526 -689766887 448763741 44226151 -951443531 426497284 -752315090 138680415 569788123 -624114671 638112697 -659322023 -910999091 -39753510 -957605534 742797604 367494041 -185375272 -858707160 812706689 56366129 -371528658 -197866854 -208539280 -53737254 -51934105 529470747 -513950225 791617320 919105826 159930210 -997210240 -130699927 -87492554 485674225 -289365227 -755830862 -37585938 426155311 -735173593 628808579 724708981 -83734492 151268325 -517913575 988678477 -813984559 -223479414 937081169 -547228935 55563050 656600519 815541044 -831628101 -719215695 -752954528 -758651216 766495648 573200055 887411987 380483378 446751670 496772296 273938962 -766651641 -804367615 893123411 -396899172 -828875408 -557678812 85690010 -32953408 -468742822 -657223972 340572457 380751242 -168800666 -608632645 -955087475 -617896936 326506317 338653927 763496794 341031639 774669703 -106807463 -523753545 -555618635 -303209208 778167536 181695479 -712860085 99168991 -122273601 684923403 632955881 -840197107 737906526 -985907964 427267563 -300616450 660257321 569927995 -279513405 -611190865 73272341 -86402862 -765370169 936600910 583621702 561807334 -970969076 615507821 -656863297 263860561 826648685 422611682 296371249 101910040 517126571 -667693507 -199125516 -924823176 423207854 110681574 525013779 975768456 -67674902 434386899 -318435967 -682738549 -224805667 -471297820 569417322 -469331244 -297366348 -946702885 222153893 736963926 740475387 18245701 -807516985 392378328 -829708996 340381007 -451084943 -213348009 904238119 -259203194 73943957 801481540 -475194540 692367713 324629628 -587568367 139473022 -494442466 155538971 -221295542 -132039872 -840560913 -217956510 363835247 -473224365 822526753 -635383075 543798035 -404421504 234000279 660733445 652732930 -363876356 928576665 996862932 -755228389 -48097973 3498588 -345605255 -869453028 -138265917 913099487 705911590 -413785823 169364859 -675057673 146914902 -985401158 -830671262 -581089288 -265882068 370000295 -382292372 936887037 718204412 85675314 -660413466 -336700715 137473507 -435986135 350413547 218625930 -294443632 976705655 -969829649 -962473329 -226888866 621165432 938287749 -87332294 520139092 -805723667 -121669206 -400935177 -914706825 550133983 315304289 -91290012 814569812 -301186890 -913760164 -148391750 658378450 -900448528 -48611133 627186984 747971748 -177724312 -515068941 -167540440 -710236381 -244218987 -712727954 659718817 796117981 -102297871 -478908838 -87833191 681855585 343728892 -842943902 858567432 -498059350 48770650 -994407382 -682321208 391114278 538573335 347483233 -87100405 294381236 -775705441 -633260973 -871482023 -674350412 259391005 53630274 -795910383 880359749 967903869 855894613 -400578827 -180914437 -342621545 -400470439 -969499037 -523080845 790507620 37940515 24064988 -902799848 -518205805 -592192782 328105231 462008962 234375360 495075420 -77781492 -891712171 966376015 -850600446 766491358 297209588 -718958184 -486488677 -526417975 352823426 -238540558 181693485 397809429 -809148265 -384587691 -85387439 -505554302 -280213044 408229576 -571575038 -770633196 -665154220 650503456 -110235110 -606767003 717887481 -684250206 288476691 -206972513 -392322044 -856198557 596320166 -370800229 -949172677 -521386965 416474644 588793354 121924911 -210251844 -167809697 -121039666 -986274435 287935831 821609308 -684786138 -409199777 -207383072 -448360304 -932777751 761660078 737107251 -799248289 439082022 -409714703 373285974 -407861828 -270853262 635231168 -457623776 262410042 -459999194 174783423 -917597644 -965105966 -988257413 852575082 -91756753 881654422 859857748 164664576 -634635187 -635367250 -295592016 -447105266 -35183573 172265709 647370041 986679446 -317804633 130207247 723525369 637537757 -798794781 -276174703 674179372 -467320882 762945129 64619821 425951024 941828110 -409677464 831768216 382170013 -717921763 248982779 212719969 -374913499 -124239027 199097299 792056124 133871587 671335918 30386789 855717386 455215151 -920774171 -938774887 796483942 251080361 718002410 -32778863 -984673022 -10936951 -345416834 -692321226 -893885596 454412353 -975673569 831289207 221931903 -902449969 105988824 -563362847 188042801 231719301 -861909653 -825810767 -651341871 526559955 -737788843 912193515 -472023271 578084071 -403445305 683947471 755695277 349470065 -219494337 884858219 -875007668 -506523777 985839043 -817753014 -580263790 126628744 245860538 435091274 -484178329 766322052 -139702169 438479499 -447315192 -779437256 -495459581 186070660 -632346192 419808400 -286395531 98427951 -615698974 -990261212 728860848 115272292 355310422 371226077 135352489 347954756 647438974 -88905295 -398726924 -276700237 578664585 -954083590 969040777 785941249 491555289 480632426 221933196 -162063403 -791846952 758591665 -868490851 50954376 -764807420 -696332723 141902958 705462432 -850565211 -9336968 -258830931 711599950 387098482 -556475681 -163290964 -334494316 -333497463 -178309686 -596626164 -235945198 640326445 931289994 -50243390 547105827 -143601041 630253468 252103557 -425545000 -282120633 -296977359 154624674 -648722316 -532516890 718941755 -239964129 70033321 -649132857 -748318974 919807930 -969754303 543267016 542746550 251374156 777649723 374792267 -321698845 -889155106 -78565930 910632369 791213726 191815821 -750706747 659167240 -585758646 -626082938 980561935 -658271518 -980797689 -552627151 918965895 -675557055 -132852807 280133995 -712356894 -197485138 109077936 -896423189 275816058 876884328 -920871175 800049350 -684623714 -401632070 -605497075 -923817325 -612797851 2065248 453856660 12544556 280195133 601624457 855839694 321108481 -824230192 -8355315 -88344364 547932751 365277760 -134695737 -301528188 -678605460 558716466 -574964026 559045093 282660575 980859616 -695226902 11080758 795769160 868523301 -211811027 573504037 658418338 576539067 572038719 744584191 194089358 -224551456 818695221 -48750060 909998738 -782275336 869269631 171179886 -543415615 841247164 225849976 -11205003 973713867 979501893 -56529037 -633684328 -217836913 390194174 168140285 -666699968 -445772240 418709802 -507492103 820091220 78007400 -725115598 260786319 -559100200 -408436858 -647373540 419483572 431538864 -751135226 -993066211 -153188271 657480323 760507339 -36405179 -416937965 -800297797 -994568965 -796709032 -447510578 307297399 -598677820 -347361291 258461584 -308958916 -827332847 455922621 316973010 -802477078 962347935 -37660676 721212864 129175995 301004074 -581539384 617388609 -357737870 190220255 399849905 445640824 374758549 175664703 482759168 -469858182 -584882371 -51814371 -796797297 338479897 298546813 -613980223 -645027772 -17264417 101090413 -447193297 33556012 -591587818 135045330 -69091455 256697904 -117008021 927646833 13492086 -225880267 -340484901 -972683789 -903093714 36686335 -189691401 -496270901 710082313 -90148756 887991174 894748507 -480954765 -66684168 273479442 -249276669 893364794 586273636 -301716985 786918381 67107075 426906044 24609606 526683671 246051570 738436277 -69926691 -256575867 774102575 640843624 -264584043 121891489 757204441 78132483 -237242769 -710721829 -406604813 -676486678 662459106 87348783 554086576 981905580 832561329 -160407528 -659144934 -657404618 429387924 469066797 173113925 797813328 -799199400 31629334 -479840304 -50750257 -644867815 -411773416 555701232 42161468 -85741977 -215969053 11410303 -531401820 713172481 -241913184 -782846304 -541941655 -749738670 -780699990 -862694445 494725682 372635853 -568641877 -251964107 42549601 -598466134 -447451943 -580348957 740124454 435328719 -905597835 860767894 566964640 -428240908 -927834569 945691521 930167260 -892145668 777567495 64817444 -86711565 327126920 -360898036 -949029653 -984610597 -34507543 746460971 -336043332 -524776702 -941995089 292654811 -349133749 -18189315 305098205 530534086 -444240910 -329389735 546670417 -216500069 659097372 -448514381 -838726116 -564802410 805190028 -868819928 678655701 -553593380 276185181 372154599 -191837030 113508367 50497647 -9283562 155052979 -196203360 -177261802 675695820 -282087844 -816259247 368893674 -95102269 -514600203 64795652 -650697070 87462847 -300014043 387898169 -620741443 -577302120 -601690853 -373316571 -355658452 -665805082 387871216 -764850581 469355751 -379210712 -986186375 -727243889 239093842 -499490984 778520450 -225642541 -578732014 -71546399 685734419 985323773 894754266 -360427947 511631268 317548281 -979571188 440952542 -172220540 777072201 983863603 553460842 -417720559 -989528911 -205036322 547621961 -567187259 -559310066 457275944 -773787541 706098610 846427228 -54783372 907799529 -840173978 -953201573 -74396989 265684056 -498230120 -181588493 -324933866 -910591838 894920865 -835208285 -398260237 540123893 607139871 225992375 -801421800 658400362 540564507 -523100003 -614729735 15707074 111706868 819082745 611718246 322778056 -523119592 -262346539 863688152 -237468205 748801376 279557110 737609822 811950695 -33530874 322732361 -815461345 283431419 575425347 -214491780 692598411 -480165930 415044266 -549135325 -349298413 -631755430 -447398866 51571937 -646723133 207652548 870165828 -999804367 -310205831 531261118 -821971460 949980736 -578310769 -528822550 633537583 -630474511 -842415920 -28159695 -785939354 -294676991 -821821761 -736459805 54469144 846965679 339498792 -60226096 408398434 -43796867 705567178 435856933 360473070 855973462 -253842234 317019650 -350676574 352422371 714932611 -105380579 -319190083 662625948 530026975 758607213 829238827 659021851 -808101542 123894391 77639862 236299735 -175524297 10695532 992621731 -71259301 328651867 948275226 -698662378 -512284544 549497569 -459535882 -376061825 821910546 434686971 -200622035 -423062760 -782890839 -420014852 -682415401 -43684165 893089828 91228570 656539101 -653640962 790775390 -12566238 52398911 -403386444 -593764976 -832817972 -537901055 -60962675 159260227 -603627223 80246195 -438193959 299060464 -647836488 -136537444 -602805910 -663945418 984229273 464065154 -398800234 -391748938 -8927143 -131751399 948085851 -804399218 -179887637 -685861114 -102932834 -973899663 560070328 -524575090 -217408628 718916192 221542605 -178971424 353394209 -98603513 735860531 918253307 595744455 546065590 -853159844 832577853 -796518378 274356986 -166411847 780649610 -402221638 -562451752 414748265 -114578411 655191922 248588197 -310774005 -400720434 -35739083 -910625403 -92971852 695410536 -581199505 -243200386 250151164 182462981 -730080663 455497528 478850689 762799046 -889060013 414954584 296405801 14864741 -92260789 -605520587 -412155148 742198927 601356514 253245208 -399510996 -527015494 -179863787 -621123806 589391992 -876328425 -558041344 807931976 425299809 -702196838 393572082 -284942061 -769925025 -379797782 -362838409 -492457032 -837138153 -303311465 468630278 763000840 919554635 373035212 -266026409 444118563 -360170950 -707339890 -569068457 387762614 28560762 -477295646 -732294095 -487144606 283188766 -143199989 566387892 -985594497 -963732637 -509811654 248302582 -573372702 635353752 3520877 -11784664 163739910 270320663 -692796001 -66284869 88848702 625754181 -121015110 551215875 118875916 186448629 114458308 -393579157 -564219614 -659174235 814325179 -307098396 -536980990 190737553 212556051 51246461 660850320 -16287236 -419381433 -635837780 627496365 -372257767 991094882 339726828 385745364 -875327424 -179120626 283786346 -388551695 522179049 422478845 -390040611 338559946 702043559 -817420003 -301108444 515835190 191676473 266128207 -76812173 364455051 681481647 488951597 168849388 318659506 -780392341 867534815 466313077 111160974 -401328569 998550173 -208151132 -713044986 -376769067 77545409 42122711 569046449 -842828351 407980935 -291734687 792164381 786190419 -899872667 -589173281 -143862511 969223425 -793595736 -422664596 484436609 724260820 950620002 -967119454 882641420 662395481 144206258 -152303310 184276991 -97665551 -245773635 -124716630 -294159274 -930464856 -33182281 -186455125 -1931846 830776212 711384083 670415429 -794145157 -637480730 -532049010 -780100119 812727941 -333310965 -967617131 -602440635 -191390404 952757204 -761038114 155790662 909138446 389147484 -292469697 627172969 150854184 -809893776 598677680 690267589 723225870 -550310675 27243900 891506713 -873280740 -668417197 -574712377 589712362 313080719 70289524 891585520 -552307650 -328316482 944113079 -339697905 258292979 -524977306 -892848631 644150941 900367036 333404002 20315699 -253943623 854180345 -25277713 475937643 477038003 -247167214 962694625 307998691 844571377 383681666 869441888 519515833 -847854010 -297012273 -324435442 964744294 -533159860 830332166 -278144282 704613410 825599148 -392028147 581843496 -519703487 -680293644 247772108 -427079672 265365051 650640032 126448905 -160283803 270672195 254364329 -146611313 662784180 -525007736 859465780 366954489 -911612677 532640519 919156331 375794720 -536570118 333126411 -254158487 238333983 -618659740 -891880637 -825328049 -146414081 14854474 -994467970 -295546252 -130220540 157822725 866614201 -163992173 -670950214 -460732780 -903674887 207346366 -153593001 -352569353 161401247 -102615442 99624766 791079230 -948394157 -761262292 898466141 -817222243 758588621 281101964 -164294174 673129162 -283643082 890774188 959526547 -393538533 924703122 347323202 -317460061 -503613990 -869932603 -371355183 -815703880 -477008880 -45342376 197344670 -585175694 -951048075 324442350 -141719575 -926472606 121335489 57734189 -562203608 -395440659 556716141 755713093 -165600341 -247995960 728631826 -768986453 -500743671 856341706 747673195 -725339595 -74049549 683065073 868541347 910845091 -543505135 308910906 675741430 313303545 194192518 -282992854 494133326 -141359071 395481357 692801068 -253928843 105341373 728968868 -425975909 481353877 -706889551 -984375410 737346196 -366061302 650954122 -951078797 -889785232 -845304949 -864263004 222545119 -868861151 -149476522 529089659 -505642594 -612359535 -232712652 -26939424 -776559660 -924508809 912223233 -958327063 71596303 -467929135 -825152073 -793634151 714269142 105607515 875925141 325629897 -25813014 -488757184 -734343701 507854062 869120976 -775413502 843892914 -860092925 -105569437 893306863 -752370172 -464885734 -505881291 718492311 -477438413 780154175 409882844 -287765725 -846638306 -147570369 -201042134 -324776330 79606777 -593786727 -418832771 -979447105 275997545 -885233416 116741648 810057109 84108919 -543385254 331552621 356909585 -930986748 405625344 -72762257 -438305003 123564414 -402227280 421650097 818718962 965003035 772333466 -37923093 -345663151 -66371266 -68143655 559099693 -530727971 -255832976 837453703 -155496534 -773271629 -491395927 211231914 -984247699 854312576 -443636844 -982994621 280454451 206342740 499623670 349666402 297496290 -272691569 -631622727 -840505482 904647338 407191031 -273662059 548061091 586107575 -719423508 808430426 -333083828 -72682352 -977506637 47355868 -316215888 -137475825 127817966 -922755206 -807312720 974936573 382765100 12410445 -555699026 -442691766 -832727395 537285354 -402739566 -946684443 140475870 -989221914 121377342 548538611 -381590142 -769079007 -894441386 -290639022 -238840319 -115324033 954541852 -645321883 15682577 -399804490 316058727 -733472408 -431903926 -13281639 -759039050 -42865601 126467798 -156134906 -364137827 728820720 -781322670 -204119791 -695546650 -588349917 258159613 544915900 -301958592 375489904 -586345793 -185859077 -881056511 -21918189 381877486 589871709 -819611834 -636308815 -453305778 637958896 251796337 -942410600 -561903283 915834782
