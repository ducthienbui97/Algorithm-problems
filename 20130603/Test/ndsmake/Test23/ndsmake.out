1159675729235
